app/model/testdata/CreateEventData.scala

```diff
@@ -50,8 +50,8 @@ object CreateEventData {
       val id = createRequest.id.getOrElse(Random.Event.id)
       val eventType = createRequest.eventType.getOrElse(Random.Event.eventType)
       val description = createRequest.description.getOrElse(Random.Event.description)
-      val location = createRequest.location.getOrElse(Random.Event.location)
-      val venue = createRequest.venue.getOrElse(Random.Event.venue)
+      val location = createRequest.location.map(l => Location(l)).getOrElse(Random.Event.location)
```

Comment:
I left it as strings so that the acceptance tests didn't have to know too much about the case class implementations